sdk/go/common/util/logging/log.go

```diff
@@ -160,6 +161,13 @@ func CreateFilter(secrets []string, replacement string) Filter {
 			continue
 		}
 		items = append(items, secret, replacement)
+
+		// Catch secrets that are serialized to JSON.
+		bs, err := json.Marshal(secret)
+		if err != nil {
+			continue
+		}
+		items = append(items, string(bs[1:len(bs)-1]), replacement)
```

Comment:
@abhinav better?

Reply:
Yeah, just one small thing: I would check for equality on original secret.
Otherwise for plain strings like "foo", you'll end up with two replacements in the filter.

```suggestion
		// Catch secrets that are serialized to JSON.
		bs, err := json.Marshal(secret)
		if err != nil {
			continue
		}
		if escaped := string(bs[1:len(bs)-1]); escaped != secret {
			items = append(items, escaped, replacement)
		}
```